cluster/src/test/scala/org/apache/spark/sql/store/ColumnUpdateDeleteTest.scala

```diff
@@ -78,7 +78,7 @@ class ColumnUpdateDeleteTest extends ColumnTablesTestBase {
     session.conf.unset(Property.ColumnBatchSize.name)
   }
 
-  ignore("SNAP-1985: update delete on string type") {
+  test("SNAP-1985: update delete on string type") {
```

Comment:
Can you please extend this test to also execute same update using query routing path (good if you also do for prepared statement)

Reply:
@vivekwiz have it as part of concurrent tests in https://github.com/SnappyDataInc/snappydata/pull/822